### PR TITLE
doc: document upgrading Ruby packages

### PR DESCRIPTION
**Pull request by alyssais**
###### Motivation for this change

As requested by @Ekleog in https://github.com/NixOS/nixpkgs/pull/51040#issuecomment-441642763.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: doc/languages-frameworks/ruby.xml [hunk 1]
```diff
@@ -50,6 +50,17 @@ bundlerEnv rec {
...
 
+ <para>
+  Updating Ruby packages can then be done like this:
+ </para>
+
+<screen>
+<![CDATA[$ cd pkgs/servers/monitoring/sensu
+$ nix-shell -p bundler --run 'bundle lock --update'
+$ nix-shell -p bundix --run 'bundix'
+}]]>
```
Comment (Ekleog) [inline doc/languages-frameworks/ruby.xml]: There is a dangling `}` brace here.

**Review by Ekleog — CHANGES_REQUESTED** [review]
Great, thank you!

File: doc/languages-frameworks/ruby.xml (hunk 1 above)
Comment (alyssais) [reply doc/languages-frameworks/ruby.xml]: Whoops, copy/paste error.

**Review by Ekleog — APPROVED** [review]

**Comment by Ekleog** [comment]
Thank you! For the record, I spotted the `}` by reading the generated documentation with `nix-build doc && firefox ./result/share/doc/nixpkgs/manual.html` :)

**Pull request merged**